hax/hax/util.py

```diff
@@ -206,9 +206,13 @@ def kv_get_raw(self, key: str, **kwargs) -> Tuple[int, Any]:
             raise HAConsistencyException('Could not access Consul KV') from e
 
     @uses_consul_cache
-    def kv_get(self, key: str, kv_cache=None, **kwargs) -> Any:
+    def kv_get(self, key: str, kv_cache=None,
+               allow_null=True, **kwargs) -> Any:
```

Comment:
I see multiple instances of `allow_null=False`, so why not instead have it as default instead of `True`?

Reply:
@mssawant There are some places where we want it to be True.
But for now number of such instances are less as compared to other.
So I will change default to False and handle cases accordingly.